### ISSUE TITLE
mask should be bool type

### ISSUE THREAD

**Issue by albonomi02** [antonioterpin/synthpix#198]
to use it with jnp.arrays like this:
a = jnp.array(...)
b = a[batch.mask]

it must be bool type.
